UnitsNet/GeneratedCode/Quantity.g.cs

```diff
@@ -376,288 +531,192 @@ public static bool TryParse([CanBeNull] IFormatProvider formatProvider, Type qua
+                case Type _ when quantityType == typeof(SpecificEntropy):
+                    return parser.TryParse<SpecificEntropy, SpecificEntropyUnit>(quantityString, formatProvider, SpecificEntropy.From, out quantity);
+                case Type _ when quantityType == typeof(SpecificVolume):
+                    return parser.TryParse<SpecificVolume, SpecificVolumeUnit>(quantityString, formatProvider, SpecificVolume.From, out quantity);
+                case Type _ when quantityType == typeof(SpecificWeight):
+                    return parser.TryParse<SpecificWeight, SpecificWeightUnit>(quantityString, formatProvider, SpecificWeight.From, out quantity);
+                case Type _ when quantityType == typeof(Speed):
+                    return parser.TryParse<Speed, SpeedUnit>(quantityString, formatProvider, Speed.From, out quantity);
+                case Type _ when quantityType == typeof(Temperature):
+                    return parser.TryParse<Temperature, TemperatureUnit>(quantityString, formatProvider, Temperature.From, out quantity);
+                case Type _ when quantityType == typeof(TemperatureChangeRate):
+                    return parser.TryParse<TemperatureChangeRate, TemperatureChangeRateUnit>(quantityString, formatProvider, TemperatureChangeRate.From, out quantity);
+                case Type _ when quantityType == typeof(TemperatureDelta):
+                    return parser.TryParse<TemperatureDelta, TemperatureDeltaUnit>(quantityString, formatProvider, TemperatureDelta.From, out quantity);
+                case Type _ when quantityType == typeof(ThermalConductivity):
+                    return parser.TryParse<ThermalConductivity, ThermalConductivityUnit>(quantityString, formatProvider, ThermalConductivity.From, out quantity);
+                case Type _ when quantityType == typeof(ThermalResistance):
+                    return parser.TryParse<ThermalResistance, ThermalResistanceUnit>(quantityString, formatProvider, ThermalResistance.From, out quantity);
+                case Type _ when quantityType == typeof(Torque):
+                    return parser.TryParse<Torque, TorqueUnit>(quantityString, formatProvider, Torque.From, out quantity);
+                case Type _ when quantityType == typeof(VitaminA):
+                    return parser.TryParse<VitaminA, VitaminAUnit>(quantityString, formatProvider, VitaminA.From, out quantity);
+                case Type _ when quantityType == typeof(Volume):
+                    return parser.TryParse<Volume, VolumeUnit>(quantityString, formatProvider, Volume.From, out quantity);
+                case Type _ when quantityType == typeof(VolumeFlow):
+                    return parser.TryParse<VolumeFlow, VolumeFlowUnit>(quantityString, formatProvider, VolumeFlow.From, out quantity);
+                default:
+                    throw new ArgumentException(
```

Comment:
If we go with ArgOutOfRangeException above, maybe we should change this to the same exception type.

Reply:
Actually... this is a TryParse. It should return false!

Reply:
Ouch :-D I didn't see, the PR review collapses too much code sometimes 😇

Reply:
Added a test that made that fail. Also updated to return false 😃